lib/resources/r4/slot_patch.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: Related Person
```

Comment:
this should be slot here and the field name as well.

Reply:
[updated](https://github.com/cerner/fhir.cerner.com/pull/608/commits/28de14e4c8417bc7009f3aea3a1ede95607d30b6#diff-8a92cfc782cbec26cd0d479d0184b0a249e166900d9f3bbef43ea04612760a9aR2)